app/models/goal.py

```diff
@@ -4,3 +4,4 @@
 
 class Goal(db.Model):
     goal_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
```

Comment:
add tasks to the Goal model with `db.relationship` this will set up the connection between goal and task.